english/teams/team-ironman.md

```diff
@@ -0,0 +1,3 @@
+Team Ironman believes that superheroes need to be looked over by a governing body that decides when their services are needed. While this may suck for superheroes 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.